server/requirements.txt

```diff
@@ -14,7 +17,7 @@ gunicorn
 humanize
 pyesbulk>=2.0.1
 PyJwt
-psycopg2-binary
+psycopg2
```

Comment:
Unfortunately, this change appears to have broken the Jenkins-based testing.

Reply:
Yeah; because in a real install it's satisfied by RPM and we don't don't try to do this at all. The original `psycopg2-binary` apparently wasn't satisfied by the RPM; but in Jenkins where we apparently don't have that RPM, we go ahead and try to build, encountering the same problem we tried to avoid by changing this dependency. Ugh.

I'm not sure how we work around this, now, short of changing the `pbench-dev` container to supersede this dependency as we do in a real install.

Reply:
(For anyone else following along at home, I'm working on building a new container for Mr. Jenkins which installs `psycopg2` from an RPM.)

Reply:
This will be fixed by #2428.

(I took the liberty of building that change, pushing it up to Mr. Jenkins, and having him retest this PR...and now it looks good.)